### PR TITLE
updated roadmap to reflect improvements planned for lakeFS hooks

### PR DESCRIPTION
**Pull request by ozkatz**

### CODE REVIEW

File: docs/understand/roadmap.md [hunk 1]
```diff
@@ -64,21 +64,47 @@ Additionally, for CD use cases, it will allow a merge operation to introduce Hiv
...
+While webhooks are easy to understand, they can be challenging in terms of operations: they require a running server listening for requests,
+ network access and authentication information need to be applied between lakeFS and the hook server and network timeouts might interfere with long running hooks.
+
+ To help reduce this burden, we plan on adding more hook types that are easier to manage - running a command line script, executing a docker image or calling out to an external orchestration/scheduling system such as Airflow or even Kubernetes. 
+
+[Track and discuss on GitHub](https://github.com/treeverse/lakeFS/issues/2231){: target="_blank" class="btn" }
+
+#### Expose merged snapshot to pre-merge hooks
+
+pre-merge hooks are a great place to introduce data validation checks. However, currently lakeFS exposes the source branch, the destination branch and the diff between them. In many cases, the desired input is actually the merged view of both branches. By have a referencable commit ID that could be passed to e.g. Spark,
```
Comment (guy-har) [inline docs/understand/roadmap.md:83]:
```suggestion
pre-merge hooks are a great place to introduce data validation checks. However, currently lakeFS exposes the source branch, the destination branch and the diff between them. In many cases, the desired input is actually the merged view of both branches. By having a referencable commit ID that could be passed to e.g. Spark,
```

**Review by guy-har — COMMENTED** [review]
LGTM

**Review by guy-har — APPROVED** [review]
LGTM

File: docs/understand/roadmap.md [hunk 2]
```diff
@@ -64,21 +64,47 @@ Additionally, for CD use cases, it will allow a merge operation to introduce Hiv
...
+#### Expose merged snapshot to pre-merge hooks
+
+pre-merge hooks are a great place to introduce data validation checks. However, currently lakeFS exposes the source branch, the destination branch and the diff between them. In many cases, the desired input is actually the merged view of both branches. By having a referencable commit ID that could be passed to e.g. Spark,
+users will be able to directly feed the merged view into a dataframe, a testing framework, etc.
 
 [Track and discuss on GitHub](https://github.com/treeverse/lakeFS/issues/1742){: target="_blank" class="btn" }
 
-### Webhook Support integration: Metastore registration
-Using webhooks, we can automatically register or update collections in a Hive/Glue metastore, using [Symlink Generation](../integrations/glue_hive_metastore.md#create-symlink), this will also allow systems that don’t natively integrate with lakeFS to consume data produced using lakeFS.
+#### Create and edit hooks directly in the UI:
```
Comment (talSofer) [inline docs/understand/roadmap.md:88]: nit; remove colon at the end of the title

**Review by talSofer — APPROVED** [review]
Looks very good!

**Pull request merged**